profileTemplate

```diff
@@ -144,11 +144,18 @@ sub isPhantom {
 }
 
 # ----------- OPTIONAL SUBROUTINE
-# If you don't want to upload all types of images you can specify the complete protocol 
-# for all types and limit the upload using the protocol names
+# This function allows the user to decide how images are to be inserted into the
+# database (violated vs. non-violated scans).
```

Comment:
I don't think this subroutine has anything to do with violated does it?

Isn't it just.. if it returns 0, the scan type isn't registered into the database?